.travis.yml

```diff
@@ -1,23 +1,14 @@
+#dist: xenial
+sudo: true
```

Comment:
Is sudo actually needed for this?

Reply:
Not needed since start using docker, removed